<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Analizador Sintactico visual</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Roberto De Santiago NÃ¡jera. 18131232</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Carlos Vazquez Aguirre 18131288</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>David Garcia Leon 19130911</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_entrada">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>199</y>
      <width>741</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>175</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Entrada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tokens</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>430</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sintactico</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_lexico">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>371</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_sintactico">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionLimpiar">
   <property name="text">
    <string>Limpiar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
